bin/fuel-core/src/cli.rs

```diff
@@ -54,7 +48,7 @@ fn init_environment() -> Option<PathBuf> {
     match dotenv() {
         Ok(path) => Some(path),
         Err(e) => {
-            error!("Unable to load .env environment variables: {e}. Please check that you have created a .env file in your working directory.");
```

Comment:
I think we should maybe not log anything, since the majority of devs won't use env files and this isn't a helpful message as the first thing they see when starting up fuel-core.

Reply:
We should probably log something to indicate when a env file is being used instead of CLI args, instead of always notifying people whenever they don't have an env file even if they dont want to use it.